.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+on:
+  pull_request:
+    branches: [develop] # If more branches are needed, add them here.
+    types: [synchronize, opened, reopened, ready_for_review]
+  push:
+    branches: [develop] # If more branches are needed, add them here.
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Ensure cache directory exists
+        run: mkdir -p /tmp/.buildx-cache
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
```

Comment:
What is the ratio of this key?

Reply:
if by ratio you mean how often this will change, it will change for each different commit (.sha)